.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
     - AUTHOR_NAME="$(git log -1 $TRAVIS_COMMIT --pretty="%aN")"
     - echo $AUTHOR_NAME
     - export TZ=Europe/Brussels
-    - npm i -g npm@6.4.1
```

Comment:
Internally, we are using npm 6.9.2. Could you please adapt? 😊

Reply:
Done ;)

Reply:
thanks